WorkloadManagementSystem/Agent/TaskQueueDirector.py

```diff
@@ -126,13 +126,13 @@
 __RCSID__ = "$Id$"
 
 import random, time, threading
-from DIRAC                                                       import S_OK, S_ERROR, List, Time, abort
+from DIRAC                                                       import S_OK, S_ERROR, gConfig, abort
```

Comment:
No this is fine. What you should not have is
import x,y,x 

See https://www.python.org/dev/peps/pep-0008/#imports